docs/building-images.md

```diff
@@ -120,3 +120,19 @@ as follows:
 Additionally, the following RPMs must be included in the RPMs directory as described above:
 * `elemental-register`
 * `elemental-system-agent`
+
+### SUSE Manager (SUMA)
+
+Automatic SUSE Manager registration can be configured for the image. The following items must be defined in the
+configuration file under the `suma` section:
+
+* `host` - this is the FQDN of the SUSE Manager host that the host needs to register against.
```

Comment:
Very small nit: the rest of the docs capitalize the first letter of the description.